Lib/fontbakery/profiles/googlefonts_conditions.py

```diff
@@ -11,6 +11,17 @@
 )
 from .shared_conditions import style
 
+
+try:
+    import google.protobuf
+except:
```

Comment:
I'll need to do some testing, but I suspect this needs to be done differently; other profiles import checks from the googlefonts profile, and only a handful of checks require this dependency. I'll look into it today. Fine to merge as-is.

Reply:
The idea here is that if a single import fails, it is evidence that the googlefonts extra was not used, so the user is instructed to do so.

Regarding other profiles cherry-picking checks from googlefonts profile, I agree that it may lead to problems if they import `googlefonts_contitions.py`, so I will double check that. Thanks for pointing this out.